MAINTAINERS

```diff
@@ -0,0 +1,11 @@
+# Maintainers should review changes promptly.
+# Github will auto-assign the below maintainers to a pull request.
+
+"**":
+    - rdimitrov
+    - kommendorkapten
+    - mnm678
+    - znewman01
```

Comment:
I think @znewman01 left as a maintainer?

Reply:
Good point. Zack should say something if that is not the case, but he is still also in the @theupdateframework/go-tuf-maintainers list.

Reply:
Right, I just copied who was in the maintainers list in the org. If that info was outdated we should booth change it there and in this file.

Reply:
Yeah that makes sense, we should use that list as authoritative.